### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by nhuntwalker**
Addressing issue #3 by introducing a workflow model. Please edit and update as suits the project.

### CODE REVIEW

File: example_dot_env [hunk 1]
```diff
@@ -1,3 +0,0 @@
-SESSION_SECRET='1845f5419299977a57a61e74c659ad804dca6f7fd5209b3888e773f123b113f8'
-
-SUNLIGHT_FOUNDATION_KEY='55a27bdc46b947c4b63b791b7cf6fa2f'
```
Comment (pascalpp) [inline example_dot_env]: @nhuntwalker if you remove this file, the line `cp example_dot_env .env` in the above docs will no longer work. I'd say leave this for now, and let's make a separate issue for removing the keys from the repo.

**Comment by nhuntwalker** [comment]
Should I close this PR and open a new one where the file still exists? Or do you want to incorporate these changes and just put the file back in

**Comment by typpo** [comment]
You should be able to update this pull request by re-adding the file and `git push`ing to the same `documentation` branch.  Then we can merge this pull request.

File: example_dot_env (hunk 1 above)
Comment (typpo) [inline example_dot_env]: To clarify my thinking on including these in the repo, these are dummy keys not used in production.  It's meant as a template for someone who wants to get their own app up and running quickly.

**Comment by nhuntwalker** [comment]
Readded the file

**Pull request merged**